SPECS/postgresql/postgresql.spec

```diff
@@ -96,6 +96,7 @@ sed -i '/DEFAULT_PGSOCKET_DIR/s@/tmp@/run/postgresql@' src/include/pg_config_man
     --with-openssl \
     --with-gssapi \
     --with-readline \
+    --with-lz4 \
```

Comment:
Please Bump the "Release" Number above by 1
Please Update the Changelog (use prior entry as an example)